_playwright-tests/UI/PopularRepo.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { describe } from 'node:test';
+import { test, type Page } from '@playwright/test';
+import { navigateToRepositories } from './helpers/navHelpers';
+import { deleteAllRepos } from './helpers/deleteRepositories';
+import { closePopupsIfExist } from './helpers/helpers';
+
+describe('Popular Repositories', () => {
+  test('Clean - Delete any current repos that exist', async ({ page }) => {
```

Comment:
I am not sure if this is a good practice to create test cases for prerequistes. It is overhead for each test case - e.g. starting browser, verifying session, navigating to console dot, loading all of it's resources, navigation to our app...

I think we should not create test cases unnecessarily. This could be easily on top of your test - if it fail your test fails as well and you will see where. Or is there any other reason why is it done separately?

Reply:
Hi, just trying to lean how multiple "tests" fit within a "describe" block. Will change that. Thank you.